apps/backend/src/modules/users/libs/types/types.ts

```diff
@@ -1,5 +1,7 @@
 export {
 	type EmailDto,
+	type FinalAnswersPayloadDto,
```

Comment:
```suggestion
	type NotificationAnswersPayloadDto,
```
Let's rename everything from Final to Notifications, that'll make more sense